### DISCUSSION TITLE
How to delete completed Tasks?

### DISCUSSION THREAD

**Discussion by ImanMontajabi** [alainm23/planify#1175] [Q&A]
I prefer not to hide completed tasks, but want to delete them permanently. How can I do this?

**Comment by alainm23** [comment]
We do not have a functionality to delete all completed tasks, but I will take this into account.

**Comment by technobirb** [comment]
Second this. I have too many minor tasks completed in inbox, now deleting them would be a chore...
#814 was marked as completed but I couldn't find the functionality in the app.

**Comment by Luflosi** [comment]
I was also looking for this and finally found it.
The functionality seems to be only accessible in the "Board" view, not in the "List" view. At least I didn't find a way to do it in the List view.

In the Board view under "Note" is "(No Section)". To the right of that are three dots. After clicking them, click "Show Completed Tasks". A window will pop up and at the bottom you can click "Delete All Completed Tasks".

**Reply by Sclarnin** [reply]
Thanks. For me, the tasks in question were within a section, and I could see the three dots in list view, but I was seeing a separate issue where completed tasks were displayed even when "show completed" was not selected. I didn't want them displayed but I didn't want to permanently delete them. When I did select "show completed" the same completed tasks then appeared as additional duplicates. Toggling to Board view and back to List view resolved the problem.
